css/css-flexbox/flex-item-transferred-sizes-padding-border-sizing.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="author" title="Sammy Gill" href="mailto:sammy.gill@apple.com" />
+    <link rel="help" href="https://drafts.csswg.org/css-sizing-4/#aspect-ratio-size-transfers" />
+    <link rel="help" href="https://drafts.csswg.org/css-sizing-3/#min-width" />
+    <link rel="help" href="https://drafts.csswg.org/css-sizing/#box-sizing" />
+    <link rel="help" href="https://bugs.webkit.org/show_bug.cgi?id=243887" />
+    <link rel="match" href="../reference/ref-filled-green-100px-square.xht" />
+  </head>
+  <style>
+    .flexContainer {
+      display: flex;
+      flex-direction: column;
+      float: left;
+    }
+    .item {
+      background: green;
+      padding-left: 25px;
+      padding-right: 25px;
+      box-sizing: border-box;
+    }
+  </style>
+  <body>
+    <div class="flexContainer">
```

Comment:
I think it would be nice to have a comment here explaining what the expectation is for the flex item.